src/types/IstioObjects.ts

```diff
@@ -139,6 +139,10 @@ export interface PodLogs {
   entries: LogEntry[];
 }
 
+export interface EnvoyProxy {
```

Comment:
I'd add in the backend a model.EnvoyProxy { Configs: interfaces{} }, just to have this mapped.

I understand that any type would be necessary if we can't map the nested types here as discussed.